test/common/schemaform/widgets/DateWidget.unit.spec.jsx

```diff
@@ -63,9 +63,11 @@ describe('Schemaform: DateWidget', () => {
     const instance = tree.getMountedInstance();
     instance.handleChange('year', 2001);
     instance.handleChange('month', '1');
-    instance.handleChange('day', '2');
```

Comment:
I thought you mentioned for this component, month is expected to be a number and day to be a string? This seems at odds with the above change in this set of tests too.